demo/EXPLOR3R/auto-drive.py

```diff
@@ -109,7 +111,7 @@ def turn():
 start()
 while not btn.any():
 
-    if ts.is_pressed:
+    if ts.is_pressed():
```

Comment:
This is actually something that changed in our most recent release, which is not included in the most recent ev3dev image by default. See https://github.com/rhempel/ev3dev-lang-python/releases/tag/0.8.0 for details on that release. If you'd like to upgrade, you can also find instructions on doing that in the README.